kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/NumberUtil.java

```diff
@@ -14,14 +14,15 @@ public static boolean isNumeric(Object value) {
   }
 
   public static float parserClusterVersion(String version) {
```

Comment:
I would add NumberFormatException to throws cause here (even it is non checked)

Reply:
sure